_docs/_latest/develop/dev/testing.md

```diff
@@ -30,6 +30,12 @@ Before you run unit tests and pylint checkers, make sure you have
 
 Install Forseti in a docker image:
 
+  ```bash
+  ./install/scripts/docker_setup_forseti.sh
+  ```
+
+Start the forseti container:
+
   ```bash
   ./install/scripts/docker_install_forseti.sh
```

Comment:
Opening issue #1996 to improve the naming quality here.

Reply:
I'll tackle that since I'm already looking at this stuff